kirkeboeger_show3/index.php

```diff
@@ -241,14 +229,18 @@ function lastOne() {
 
 
 
-<div id="openseadragon1" style="width: site-width; height:650px; border-top: 1px solid #f6f6f6;background-color:#666666";"margin: auto;></div>
+<div id="openseadragon1" style="width: site-width; height:650px; border-top: 1px solid #f6f6f6;background-color:#666666;margin: auto;"></div>
 
 
 
 
 
 <!-- OSD -->
-+<script src="https://cdn.jsdelivr.net/npm/openseadragon@2.3/build/openseadragon/openseadragon.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.min.js"
```

Comment:
I thought I might need jQuery, but didn't end up needing it... I'm leaving it in though, because it may make things easier for you for future additions.